shared/src/vm/prefix_iter.rs

```diff
@@ -54,14 +51,14 @@ where
 {
     fn default() -> Self {
         Self {
-            index: PrefixIteratorId::new(0),
+            index: PrefixIteratorId::default(),
             iterators: HashMap::new(),
```

Comment:
Very pedantic, but it would look more consistent to also use `HashMap::default()`

Reply:
done, thanks!